src/dashboard/src/components/api/views.py

```diff
@@ -519,6 +498,8 @@ def get_levels_of_description(request):
     response = [{l.id: l.name} for l in levels]
     return helpers.json_response(response)
 
+# TODO should this have auth?
+@_api_endpoint(expected_methods=['GET'])
 def fetch_levels_of_description_from_atom(request):
```

Comment:
This had no auth or verb restriction previously - needs to be tested to see if it still works with external consumers.